azkaban-common/src/main/java/azkaban/imagemgmt/daos/ImageVersionDaoImpl.java

```diff
@@ -372,6 +376,22 @@ public void updateImageVersion(final ImageVersion imageVersionRequest)
     }
   }
 
+  @Override
+  public boolean isInvalidVersion(final String imageTypeName, final String version)
+      throws ImageMgmtException {
+    try {
+      return this.databaseOperator.query(
+          SELECT_INVALID_IMAGE_VERSION_QUERY, new CheckInvalidImageVersionHandler(),
+          imageTypeName.toLowerCase(), version, State.NEW.getStateValue(),
+          State.ACTIVE.getStateValue());
+    } catch (final SQLException ex) {
+      log.error("Exception while fetching image version ", ex);
```

Comment:
Nit: Exception while verifying whether image version is invalid or not.

Reply:
Done.